spec/model/kubernetes/kubernetes_cluster_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative "../spec_helper"
+
+RSpec.describe KubernetesCluster do
+  subject(:kc) {
+    described_class.new(
+      name: "kc-name",
+      kubernetes_version: "v1.32.0",
+      location: "hetzner-fsn1",
+      cp_node_count: 3
+    ) { _1.id = "3ebf43d3-118a-4af7-87d2-59a3b9bf9116" }
```

Comment:
Is setting the `id` actually needed here?  You aren't saving the object, and it doesn't look like either of the two specs would need to access the id.